frontend/templates/contraceptives/tabs/Effect.tsx

```diff
@@ -0,0 +1,79 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+import { Column, Description, Row } from './StyledComponents';
+
+// Styles
+
+const Container = styled(Column)`
+  align-items: center;
+  text-align: center;
+  width: 30%;
+`;
+
+const Label = styled(Description)`
+  margin-left: 1rem;
+`;
+
+const RowWrapped = styled(Row)`
+  flex-wrap: wrap;
+  justify-content: space-between;
+  row-gap: 1rem;
+`;
+
+const SideEffectsIconContainer = styled.div`
+  align-items: center;
+  display: flex;
+  height: 6vh;
+//  height: 5px;
+`;
+
+const SideEffectsLabel = styled(Description)`
+  margin: 0;
+`;
+
+// Components
+
+export declare interface EffectProps {
+  benefitsInfos: [ReactElement, string][],
+  sideEffectsInfos: [ReactElement, string][],
+};
+
+const Effect = ({
+  benefitsInfos,
+  sideEffectsInfos,
+ }: EffectProps) => {
+  return (
+    <>
+      <h3>Non-contraceptive benefits</h3>
+      <Column>
+        {benefitsInfos.map((infos) => {
```

Comment:
It seems that on the last benefit, since it is two lines, the div gets bigger vertically, but the icon still is at to the top of the div. We should probably center the icon within the div vertically

<img width="515" alt="Screen Shot 2021-11-07 at 7 47 29 PM" src="https://user-images.githubusercontent.com/17476012/140668792-1779227a-38c1-42bd-a027-6a13feb8ced3.png">